cli/command/manifest/create_list.go

```diff
@@ -0,0 +1,82 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/distribution/reference"
+	"github.com/docker/docker/registry"
+)
+
+type annotateOpts struct {
+	amend bool
+}
+
+func newCreateListCommand(dockerCli command.Cli) *cobra.Command {
+
+	opts := annotateOpts{}
+
+	cmd := &cobra.Command{
+		Use:   "create newRef manifest [manifest...]",
+		Short: "Create a local manifest list for annotating and pushing to a registry",
+		Args:  cli.RequiresMinArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return createManifestList(dockerCli, args, opts)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVarP(&opts.amend, "amend", "a", false, "Amend an existing manifest list transaction")
+	return cmd
+}
+
+func createManifestList(dockerCli command.Cli, args []string, opts annotateOpts) error {
+
+	// Just do some basic verification here, and leave the rest for when the user pushes the list
```

Comment:
Inline comments are not great for documenting this type of thing. It's never clear where the scope of this comment ends. As code is changed it becomes even less clear.

Could this be split out into a separate function? The comment may not be necessary if the function name is descriptive enough, or the comment can become a godoc string on the function instead (which makes the scope obvious).